edge/pkg/metamanager/client/endpoint.go

```diff
@@ -0,0 +1,107 @@
+package client
+
+import (
+	"encoding/json"
+	"fmt"
```

Comment:
please arrange the imports.
Applicable to similar places

Reply:
done